code/__DEFINES/jobs.dm

```diff
@@ -107,6 +107,7 @@
 #define F13PROSPECTOR	(1<<4)
 #define F13DEPUTY		(1<<5)
 #define F13MAYOR		(1<<6)
+#define F13SECRETARY	(1<<6)
```

Comment:
Not exactly sure what you'd call it, but you see the number thing next to the role? Yeah, that's wrong. It's supposed to be 7. And even after you changed that, you have to update the numbers (basically increasing them by 1) for the rest of the Oasis roles after the Secretary role to fit in now.

Reply:
alright
was just following the den doctors update as a template which did the same thing
fixing it now

Reply:
didnt do anything, linters still stuck

Reply:
Hm, alright. On lines 94, 98, and 120, there are unnecessary commas which ends lists. I thiiiiink that breaks stuff. Try removing those and see if it works.

Reply:
still nothing

also, what does this mean?
its the only error i can see in the logs?
Run yogstation13/DreamAnnotate@v1
Error: ENOENT: no such file or directory, open 'output-annotations.txt'

Reply:
Okay, after digging through some PRs and their checks, it seems that you didn't indent properly somewhere in code/modules/jobs/job_types/oasis.dm. Not sure where yet, but I'm looking.

Reply:
yeah, accidentally made it worse trying to look for the bad indent

Reply:
gonna try and re-do this from scratch